homeassistant/components/fully_kiosk/coordinator.py

```diff
@@ -24,16 +24,16 @@ def __init__(self, hass: HomeAssistant, entry: ConfigEntry) -> None:
             entry.data[CONF_HOST],
             DEFAULT_PORT,
             entry.data[CONF_PASSWORD],
-            use_ssl=entry.data[CONF_SSL],
-            verify_ssl=entry.data[CONF_VERIFY_SSL],
+            use_ssl=entry.data.get(CONF_SSL, False),
+            verify_ssl=entry.data.get(CONF_VERIFY_SSL, False),
         )
         super().__init__(
             hass,
             LOGGER,
             name=entry.data[CONF_HOST],
             update_interval=UPDATE_INTERVAL,
         )
-        self.use_ssl = entry.data[CONF_SSL]
+        self.use_ssl = entry.data.get(CONF_SSL, False)
```

Comment:
If you move this above the `self.fully` you only need to do a `.get` for this value once